Python/bytecodes.c

```diff
@@ -193,7 +191,21 @@ dummy_func(
             ERROR_IF(res == NULL, error);
         }
 
-        inst(BINARY_OP_MULTIPLY_INT, (left, right -- prod)) {
+        family(binary_op, INLINE_CACHE_ENTRIES_BINARY_OP) = {
+            BINARY_OP,
+            BINARY_OP_ADD_FLOAT,
+            BINARY_OP_ADD_INT,
+            BINARY_OP_ADD_UNICODE,
+            BINARY_OP_GENERIC,
+            // BINARY_OP_INPLACE_ADD_UNICODE,  // This is an odd duck.
```

Comment:
Why is this commented out? We need to generate tables from this information, so it has to be complete.

Reply:
Because it behaves like a super-instruction and its stack effect is different from the others in its family, which makes it fail the check on consistent family members in the generators. I have an idea on how to solve this (make it an _actual_ super-instruction) but that has to wait until after I've refactored the generator and done some other work first (introducing macros and ops).

Reply:
Either implement family checks fully, or leave them for another PR. I don't like half implementing the checks and commenting out this instruction.